packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -158,7 +158,7 @@ class HotRunner extends ResidentRunner {
     }
   }
 
-  void _addBenchmarkData(String name, int value) {
+  void addBenchmarkData(String name, int value) {
```

Comment:
Maybe add `@visibleForTesting` and/or `@protected` since this seems to only be exposed to override it in a test?

Reply:
I have undone changes to `_addBenchmarkData` and instead opted to inject it instead. Seems cleaner than exposing this random method as a public one. 

Can you take a quick look again?